modules/nf-core/mapad/map/meta.yml

```diff
@@ -0,0 +1,81 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/yaml-schema.json
+name: "mapad_map"
+description: Map short-reads to an indexed reference genome
+keywords:
+  - mapad
+  - short-read
+  - align
+  - reference
+  - fasta
+  - bam
+  - map
+  - fastq
```

Comment:
can probably add 'ancient DNA' or 'adna' or something somewhere, check https:nf-core/modules to see if there are any precendents

Reply:
Maybe also add `mapping`?